src/SFA.DAS.EmployerIncentives.Abstractions/Domain/AggregateRoot.cs

```diff
@@ -4,4 +4,8 @@ public abstract class AggregateRoot<IdType, EntityModel> : Entity<IdType, Entity
     {
         protected AggregateRoot(IdType id, EntityModel properties, bool isNew) : base(id, properties, isNew) { }        
     }
+
+    public abstract class AggregateRoot
```

Comment:
An aggregate root is an entity so it should have an identity.  This doesn't enforce that hence the object inheriting from it is probably a value object.  Value objects should contain most of the business logic so that would fit with NewApprenticeIncentive.